lib/datadog/tracing/writer.rb

```diff
@@ -185,6 +213,28 @@ def reset_stats!
         @traces_flushed = 0
         @transport.stats.reset!
       end
+
+      def too_many_requests?(response)
```

Comment:
If we have a `SendResult` struct with `too_many_requests?` method, why do we need another `too_many_requests?` method here?